CRM/Contribute/Page/ContributionPage.php

```diff
@@ -675,7 +675,7 @@ public function pagerAtoZ($whereClause, $whereParams) {
 SELECT DISTINCT UPPER(LEFT(title, 1)) as sort_name
 FROM civicrm_contribution_page
 WHERE $whereClause
-ORDER BY LEFT(title, 1)
+ORDER BY UPPER(LEFT(title, 1))
```

Comment:
It's because to make it compatible with the same column we included in the select clause. If not, it errors with the message(`ORDER BY clause is not in SELECT list`). Ordering with alias name could work here to avoid the query look fishy, will be updating this in next PR.